CHANGES.md

```diff
@@ -29,8 +29,7 @@
 
 ### Documentation
 
-<!-- Major changes to documentation and policies. Small docs changes
```

Comment:
This comment shouldn't be removed, in case you saw me removing it from my PR and then did the same, I did that by mistake and will be reverting it.